src/test/resources/journeys.conf

```diff
@@ -35,6 +35,7 @@ journeys {
       login,
       home
     ]
+    run-if = ["label-A"]
```

Comment:
run-if and skip-if are not mandatory, can we leave the first example without it?